lib/loadable_config.rb

```diff
@@ -28,7 +28,7 @@ def self.attribute(attr, type: :string, optional: false)
     attr_accessor attr
 
     singleton_class.instance_eval do
-      delegate attr, to: :instance
+      define_method(attr) { self.class.send(attr) }
```

Comment:
`expect(config_class.instance.number).to eq 1` 

This passes. What would be a failing example?